### ISSUE TITLE
Install as devDependency

### ISSUE THREAD

**Issue by mhofman** [GoogleChromeLabs/jsvu#69]
I'd like to use `jsvu` to help run some tests on various engines for a package I'm writing.
For that I wouldn't install `jsvu` globally but instead would have it as a devDependency.

In that case I want `jsvu` to install engines in a specific folder inside the project's root, instead of in the user's home.

**Comment by mathiasbynens** [comment]
Could you unpack this use case a little bit for me?

The unstated assumption in your request is that jsvu-installed binaries suffer from the same versioning/collision problems that npm packages do, but that’s not the case.

If two local projects I’m working on both depend different versions of `lodash`, then of course the project-specific version must be installed within the relevant project directory’s `node_modules` folder.

jsvu-installed binaries however are always “the latest version” by default, or explicitly versioned in their binary name in case of [a specific engine version being installed](https://github.com/GoogleChromeLabs/jsvu#installing-specific-versions). That is:

```
# This is the “latest” V8 version, updated whenever `jsvu` runs.
~/.jsvu/v8
# This is V8 v7.4.288, as installed by running `jsvu v8@7.4`.
~/.jsvu/v8-7.4.288
```

Would your project depend on the latest available engine versions? In that case, it seems fine to install globally to `~/.jsvu/v8`, since that’s where other projects could also expect the latest V8 binary to live.

Or would your project depend on specific engine versions, e.g. V8 v7.4.288? In that case, it also seems fine to install globally to `~/.jsvu/v8-7.4.288`, since no project would refer to that path while expecting a different version.

**Comment by mhofman** [comment]
I guess my main issue is a conceptual one. If my package uses jsvu internally, it feels weird for it to rely on external files.

One impact of that is what if the user doesn't have jsvu installed, in particular didn't setup it's `$PATH`.\
I guess installing as a devDependency, and invoking `jsvu v8@7.4 && ~/.jsvu/v8-7.4.288` in an npm script would do the trick.

**Comment by mathiasbynens** [comment]
> One impact of that is what if the user doesn't have jsvu installed, in particular didn't setup its `$PATH`.
> I guess installing as a `devDependency`, and invoking `jsvu v8@7.4 && ~/.jsvu/v8-7.4.288` in an npm script would do the trick.

Exactly! Alternatively, you could set the `PATH` as part of the `npm script`. (It would only alter the `PATH` for that particular command.)
